libpod/events.go

```diff
@@ -37,6 +37,24 @@ func (c *Container) newContainerEvent(status events.Status) {
 	}
 }
 
+// containerRenameEvent creates a new event if a container is renamed
+func (c *Container) containerRenameEvent(status events.Status) {
+	e := events.NewEvent(status)
+	e.ID = c.ID()
+	e.Name = c.Name()
+	e.Image = c.config.RootfsImageName
+	e.Type = events.Container
+
+	e.Details = events.Details{
+		ID:         e.ID,
+		Attributes: c.Labels(),
+	}
+
+	if err := c.runtime.eventer.Write(e); err != nil {
+		logrus.Errorf("Unable to write pod event: %q", err)
+	}
+}
```

Comment:
The function name and content do not match!

Why even create the same function as `newContainerEvent`?
just call `newContainerEvent(rename)`

Reply:
I had intentions to  populate and add new renamed field but decided to not add it later, I guess yes using `newContainerEvent` is better.